app/javascript/components/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useSelector } from 'react-redux';
+import { getGreeting } from './Greeting';
+
+const App = () => {
+  const greet = useSelector((state) => state.greetings);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getGreeting());
+  }, [dispatch]);
+  
+  return (
+  <>
+    <h1 style={{textAlign: 'center'}}>Random Greeting Generator: </h1>
+    <h2 style={{textAlign: 'center', fontWeight: 'bold'}}>{greet}</h2>
+  </>
+  );
+};
```

Comment:
- [ ] You did a great job working on this milestone 💯. However, according to the requirements:

![image](https://github.com/HunterX-7/Hello-World/assets/77069011/edaa83ad-781e-42ed-89b8-57deee8dc116)

Please make sure to use this `<App>` component to setup your routes, in this case, the one that is going to display the `<Greeting>` component. 🤓 🔧